apps/pie-storybook/stories/pie-link.stories.ts

```diff
@@ -13,19 +14,28 @@ type LinkProps = SlottedComponentProps<LinkBaseProps>;
 type LinkStoryMeta = StoryMeta<LinkProps>;
 
 const defaultArgs: LinkProps = {
+    tag: 'a',
     variant: 'default',
     size: 'medium',
     href: 'https://pie.design',
-    target: '_target',
+    target: '_blank',
```

Comment:
target is an optional attribute, and this is only for storybook as I thought it's cooler to open the link in a new tab, otherwise, the story iframe will be replaced by the docs site 🤣